### DISCUSSION TITLE
How do I get prettier to work in LunarVim?

### DISCUSSION THREAD

**Discussion by HarrisonHemstreet** [LunarVim/LunarVim#2791] [Q&A] [answered]
I'm currently trying to get LunarVim to work with Prettier. The documentation within the LunarVim website is greatly appreciated, though it has left me confused due to my lack of experience with configuring NeoVim.

What is the best way to enable Prettier with LunarVim so that when I save (`:w`) my JavaScript files, they will auto format to my specs I have set?

**Comment by HungLe14** [comment, marked as answer]
@HarrisonHemstreet 

I'm using it to format my javascript file. I think you can use it.
```lua
local formatters = require "lvim.lsp.null-ls.formatters"
formaters.setup{
    { command = "black" },
    {
        command = "prettier",
        filetypes = { "javascript" , "javascriptreact"}
     }
}
```

**Reply by HarrisonHemstreet** [reply]
Thank you! Where do I put this?

**Reply by HungLe14** [reply]
You can put it in config.lua file
